packages/subgraph/src/mappings/superToken.ts

```diff
@@ -447,3 +450,14 @@ function _createTransferEventEntity(event: Transfer): void {
     ev.token = event.address;
     ev.save();
 }
+
+export function handleApproval(event: Approval): void {
+    const flowOperatorID = getFlowOperatorID(event.params.spender, event.address, event.params.owner);
+    const flowOperator = FlowOperator.load(flowOperatorID);
+    if (!flowOperator) return;
```

Comment:
As we discussed it's only for vesting/flow allowances given (when transfer cliff amount), not for autowrap. So we can't store all approval as FlowOperators that's why there is a check if the flow operator exists then only update the `allowance`.

if there wasn't an ACL event before -  it will skip here and handle allowance in `if(!getOrInitFlowOperator) and update its correct state into the entity.

Reply:
I don't recall discussing this.

I'd say the entity should always be created when ERC-20 allowance is given. This might warrant an entity name change for more clarity (hence the talk about "access settings" in the issue).